blog/en/blog/2025/03/31/march-monthly-report.md

```diff
@@ -0,0 +1,164 @@
+
+This feature is especially beneficial for two reasons:
+
+- The JWT can be removed from request attributes when `hide_credential = true`, this offers a secure alternative for token passing without exposure.
+
+- It prevents custom plugins from duplicated code, for example, retrieve and parse JWT objects.
+
+### 6. Add JWT Audience Validator in `openid-connect` Plugin
+
+PR: https://github.com/apache/apisix/pull/11987
+
+Contributor: [bzp2010](https://github.com/bzp2010)
+
+Add JWT audience authentication to the `openid-connect` plugin to:
+
+- Allow the configuration of audience claim to enforce validation of JWT Audience Validator.
+- Asserts that it should be equal to or contain the `client_id` to comply with the OIDC specification requirements; otherwise, rejects the request.
+- Allows customization of the claim.
+
+Directly implemented in plugin code since jwt-validators only supports local verification, not the Introspection API. Features are disabled by default for compatibility. Users can enable them as needed.
+
+### 7. Set Default Value of `ssl_trusted_certificate` to `system`
+
+PR: https://github.com/apache/apisix/pull/11993
+
+Contributor: [Revolyssup](https://github.com/Revolyssup)
+
+When testing AI plugins, we found that `ssl_trusted_certificate` must be set to `system`; otherwise, APISIX reports "unable to get local issuer certificate" when accessing external AI services.
+
+This PR:
+
+- Moves schema validation into `read_yaml_conf` for consistency, with `local_conf` now just calling `read_yaml_conf` internally and adding caching.
+- Ensures schema validation happens first when reading YAML files, setting the default trusted certificate value to `system` before overrides (including replacing `system` with certificate paths).
+- Removes support for combining multiple certificates.
+
+### 8. Add `valid_issuers` Field in `openid-connect` Plugin
+
+PR: https://github.com/apache/apisix/pull/12002
+
+Contributor: [Revolyssup](https://github.com/Revolyssup)
+
+Adds a field `valid_issuers` when JWKs is used to verify the issuer of the JWT. Whitelist the vetted issuers of the JWT. When not passed by the user, the issuer returned by the discovery endpoint will be used. If both are missing, the issuer will not be validated.
+
+### 9. Implement Rate-Limiting based on Fallback Strategy in `ai-proxy-multi` Plugin
+
+PR: https://github.com/apache/apisix/pull/12047
+
+Contributor: [shreemaan-abhishek](https://github.com/shreemaan-abhishek)
+
+In a previous refactor PR, the fallback strategy was removed to accommodate the `ai-rate-limiting` plugin. This PR introduces a fallback strategy that fits well with the rate-limiting construction in the `ai-proxy-multi` plugin.
+
+### 10. Support Embeddings API in `ai-proxy` Plugin
+
+PR: https://github.com/apache/apisix/pull/12062
+
+Contributor: [shreemaan-abhishek](https://github.com/shreemaan-abhishek)
+
+This PR adds support for proxying the embeddings API in the `ai-proxy` plugin.
```

Comment:
```suggestion
This PR adds support for proxying to the embedding models in the `ai-proxy` and `ai-proxy-multi` Plugins.
```

a lot for changes for `ai-proxy` are common denominators - that also change for the multi plugin